lib/Listener/CalDavEventListener.php

```diff
@@ -113,24 +113,46 @@ public function handle(Event $event): void {
 			return;
 		}
 
+		if ($this->roomService->hasExistingCalendarEvents($room, $this->userId, $vevent->UID->getValue())) {
+			$this->roomService->setObject($room);
```

Comment:
*nit*: I was a bit confused that it's called setObject when you don't set anything.
Should we add a `resetObject()` method or explicitly calling it with:
```suggestion
			$this->roomService->setObject($room, '', '');
```

Reply:
lol I added that but didn't remember to use it. Changed.